src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -588,26 +588,19 @@ has to be updated, then for MEG it could be considered to be a new session.
 
 ## Empty-room MEG recordings
 
-Empty-room MEG recordings capture the environmental and recording system's
-noise.
-In the context of BIDS it is RECOMMENDED to perform an empty-room recording for
-each experimental session.
-It is RECOMMENDED to store the empty-room recording inside a subject directory
-named `sub-emptyroom`.
-The label for the [`task-<label>`](../99-appendices/09-entities.md#task) entity in the empty-room recording SHOULD be
-set to `noise`.
-If a [`session-<label>`](../99-appendices/09-entities.md#ses) entity is present, its label SHOULD be the date of the
-empty-room recording in the format `YYYYMMDD`, that is `ses-YYYYMMDD`.
-The `scans.tsv` file containing the date and time of the acquisition SHOULD
-also be included.
-The rationale is that this naming scheme will allow users to easily retrieve the
-empty-room recording that best matches a particular experimental session, based
-on date and time of the recording.
-It should be possible to query empty-room recordings just like usual subject
-recordings, hence all metadata sidecar files (such as the `channels.tsv`) file
-SHOULD be present as well.
+Empty-room MEG recordings capture the environmental and recording system's noise.
+In the context of BIDS it is RECOMMENDED to perform an empty-room recording for each experimental session.
```

Comment:
Do really recommend to acquire noise recordings before each **session**, i.e. a subject does 3 runs x 3 tasks, you record 9 noise sessions? Or does "experimental session" refer to "all the contiguous sessions of a subject"?
Is this how you do it at the Donders?

I had the impression that at many sites, this is more of a daily thing, done by the lab manager also as a way to track the technical health of the MEG system.

Reply:
yes, would be important to be precise here also with regards to what ["session" means in BIDS](https://bids-specification.readthedocs.io/en/latest/02-common-principles.html#definitions)

Reply:
> Do really recommend to acquire noise recordings before each **session**, i.e. a subject does 3 runs x 3 tasks, you record 9 noise sessions?

My understanding is that one "session" is one visit to the lab

Reply:
> My understanding is that one "session" is one visit to the lab

Yes, obviously. I got distracted... Please ignore this previous comment about the session.